peachjam/templates/peachjam/_timeline_events.html

```diff
@@ -0,0 +1,60 @@
+{% load peachjam %}
+{% load peachjam i18n %}
+
+
+  <div class="tab-pane fade" id="history-tab" role="tabpanel" aria-labelledby="history-tab">
+      <div class="card">
+        <div class="card-body">
+          <h4>History of this document</h4>
+          {% if timeline_events %}
+            <div class="vertical-timeline">
+              {% for history_item in timeline_events %}
+                <div class="vertical-timeline__item">
+                  <div class="card">
+                    <div class="card-header">
+                      <h5 class="mb-0">
+                        {{ history_item.date|parse_string_date|date:"d F Y" }}
+                        {% if history_item.date == current_object_date %}
+                          <span class="badge rounded-pill bg-primary">this version</span>
+                        {% endif %}
+                      </h5>
+                      {% for event in history_item.events %}
+                        <div>
```

Comment:
This must be in `.card-body` which comes after `.card-header`.

Reply:
refactored with nick